logstash-core/spec/api/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 # encoding: utf-8
 API_ROOT = File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "lib", "logstash", "api"))
 
+require "stud/task"
```

Comment:
What code is using this? I don't see any code in the PR using stud. 

Is there a missing require at the top of a .rb file somewhere? We probably shouldn't put requires in the spec_helper for libraries used by app code.